components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1589 @@
+
+### Vertical Discretization
+
+The previous equation set [](#vh-mass) to [](#vh-momentum) is for an arbitrary layer bounded by $z^{\text{top}}$ above and $z^{\text{bot}}$ below. 
+We now provide the details of the vertical discretization. 
+The ocean is divided vertically into $K_{max}$ layers, with $k=1$ at the top and increasing downwards (opposite from $z$).
+Layer $k$ is bounded between $z_k^{\text{top}}$ above and $z_{k+1}^{\text{top}}$ below.
+
+The layer thickness of layer k, used in MPAS-Ocean, is
+
+$$
+h_k = \int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} dz.
+$$ (def-thickness)
+
+In Omega we will use the pseudo-thickness,
+
+$$
+{\tilde h}_k(x,y,t) = \int_{{\tilde z}_{k+1}^{\text{top}}}^{{\tilde z}_k^{\text{top}}} d{\tilde z}
+= \frac{1}{\rho_0} \int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho dz,
+$$ (def-pseudo-thickness)
+
+which is the mass per unit area in the layer, normalized by $\rho_0$. The density-weighted average of any variable $\phi({\bf x},t)$ in layer $k$ is
+
+$$
+{\overline \phi}^{\tilde{z}}_k(x,y,t) = 
+\frac{\int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho \phi dz}
+     {\int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho dz}
+     =
+\frac{\frac{1}{\rho_0}\int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho \phi dz}
+     {\frac{1}{\rho_0}\int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho dz}
+     =
+\frac{\int_{{\tilde z}_{k+1}^{\text{top}}}^{{\tilde z}_k^{\text{top}}} \phi d{\tilde z}}
+     {\int_{{\tilde z}_{k+1}^{\text{top}}}^{{\tilde z}_k^{\text{top}}} d{\tilde z}}
+=
+\frac{1}{{\tilde h}_k}\int_{{\tilde z}_{k+1}^{\text{top}}}^{{\tilde z}_k^{\text{top}}} \phi d{\tilde z}.
+$$ (def-layer-average)
+
+Rearranging, is it useful to note that
+
+$$
+\frac{1}{\rho_0}\int_{z_{k+1}^{\text{top}}}^{z_k^{\text{top}}} \rho \phi dz
+ = 
+{\tilde h}_k {\overline \phi}^{\tilde{z}}_k(x,y,t).
+$$ (h-phi)
+
+
+### Layered Tracer & Mass 
+
+Dividing the tracer equation [](#vh-tracer) for layer $k$ by the normalization constant $\rho_0$, and substituting [](#def-pseudo-velocity) and [](#h-phi), we have
+
+$$
+\frac{d}{dt} \int_{A} {\tilde h}_k {\overline \varphi}^{\tilde{z}}_k  \, dA
++
```

Comment:
@xylar, mentions this below, but the $\frac{d}{dt}$'s should be clarified throughout. Since $A$ isn't time dependent, $\frac{d}{dt}\int_A f dA = \int_A\frac{\partial f}{\partial t} dA$